app.js

```diff
@@ -23,6 +23,7 @@ const expressWinston = require('express-winston');
 const session = require('express-session');
 const PostgresStore = require('connect-session-sequelize')(session.Store);
 const responses = require('./api/responses');
+const passport = require('./api/services/passport');
```

Comment:
`passport` was used everywhere, so I just moved it here for use in the main app, rather than explicitly adding it it each router.